### PR TITLE
feat(banner): include pattern

### PR DESCRIPTION
**Pull request by kalinchernev**

### CODE REVIEW

File: framework/atoms/blockquotes/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Blockquote
```
Comment (degliwe) [inline framework/atoms/blockquotes/README.md]: Blockquotes

**Comment by yhuard** [comment]
@kalinchernev just so you know, when this PR will be merged I may move the blockquote atom under the "typography" collection 😉

**Comment by kalinchernev** [comment]
@yhuard Could be done before that, consider it as WIP

**Comment by yhuard** [comment]
Alright! Do you mind moving it now? 😃

File: framework/atoms/typography/blockquotes/package.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@ec-europa/europa-component-library-blockquotes",
```
Comment (yhuard) [inline framework/atoms/typography/blockquotes/package.json]: Should be `@ec-europa/europa-component-library-typography-blockquotes`

File: scripts/copy.js [hunk 3]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable */
```
Comment (yhuard) [inline scripts/copy.js]: May I ask why?

Comment (kalinchernev) [inline scripts/copy.js]:
```
/home/kalin/projects/apps/europa-component-library/scripts/copy.js
   9:12  warning  Unexpected console statement  no-console
  12:10  warning  Unexpected console statement  no-console
  18:12  warning  Unexpected console statement  no-console
  21:10  warning  Unexpected console statement  no-console
```
prevents me from push

**Comment by yhuard** [comment]
Good work guys!

**Pull request merged**